mon-pix/tests/acceptance/fill-in-campaign-code_test.js

```diff
@@ -131,4 +131,72 @@ module('Acceptance | Fill in campaign code page', function (hooks) {
       });
     });
   });
+
+  module('on international domain (.org)', function () {
+    module('when connected', function () {
+      module('when accessing the fill in campaign code page with "Français" as default language', function () {
+        test('does not display the language switcher', async function (assert) {
+          // given & when
+          await authenticate(user);
+          const screen = await visit('/campagnes');
+
+          // then
+          assert.strictEqual(currentURL(), '/campagnes');
+          assert.dom(screen.getByRole('button', { name: 'Accéder au parcours' })).exists();
+        });
```

Comment:
Ce test ne semble pas faire ce qu'il dit (`does not display the language switcher`).

Reply:
J'ai retirer le code qui devait tester que le sélecteur ne s'affiche pas, je n'ai pas fais attention. Dans les autres tests, il n'y a pas besoin de vérifier la présence parce c'est testé en intégration (le bon fonctionnement).